integration_tests/fixtures/forseti/main.tf

```diff
@@ -124,25 +124,24 @@ resource "random_pet" "random_name_generator" {
 }
 
 resource "google_kms_key_ring" "test-keyring" {
-  project = "${var.project_id}"
-  name = "keyring-${random_pet.random_name_generator.id}"
+  project  = var.project_id
+  name     = "keyring-${random_pet.random_name_generator.id}"
   location = "global"
 }
 
 resource "google_kms_crypto_key" "test-crypto-key" {
   name            = "crypto-key-${random_pet.random_name_generator.id}"
-  key_ring        = "${google_kms_key_ring.test-keyring.self_link}"
+  key_ring        = google_kms_key_ring.test-keyring.self_link
   rotation_period = "100000s"
 }
 
 provider "gsuite" {
-  version     = "~> 0.1"
-  credentials = "/workspace/release-silver.json"
-  impersonated_user_email = "admin@silver.forsetisecurity.dev"
+  version                 = "~> 0.1"
+  impersonated_user_email = var.gsuite_admin_email
```

Comment:
Main functional change is to remove the credentials variable here because it is not needed, and use a variable for the impersonated user email. Same thing with the test group that is created, this will be created in the org being tested.

Reply:
Nice!